ext_emconf.php

```diff
@@ -6,21 +6,21 @@
 
 $EM_CONF[$_EXTKEY] = [
 	'title' => 'Grids for bootstrap',
-	'description' => 'Gridelements for bootstrap v4. Column grids, tabs and accordion.',
+	'description' => 'Gridelements for bootstrap v5. Column grids, tabs and accordion.',
 	'category' => 'misc',
 	'author' => 'Pascal Mayer',
 	'author_email' => 'typo3@lascap.ch',
 	'author_company' => '',
-	'version' => '2.2.0',
+	'version' => '3.0.0',
 	'state' => 'stable',
 	'uploadfolder' => '0',
 	'createDirs' => '',
 	'modify_tables' => '',
 	'clearCacheOnLoad' => 1,
 	'constraints' => [
 		'depends' => [
-			'typo3' => '10.4.0-10.4.99',
-			'gridelements' => '10.0.0-10.99.99',
+			'typo3' => '10.4.0-11.5.99',
+			'gridelements' => '11.0.0-10.99.99',
```

Comment:
First of all. Thank you for that change ! <3  I am currently thinking of using your branch for an update on one of my projects.

Second: I think this line should be changed. On composer.json you depend on 9, 10 & 11. I guess this here should rather be: `9.0.0-11.99.99`

Reply:
@RozbehSharahi thanks, I think you are right. Please make that change.

Reply:
OK, will check if I can do this this weekend. Maybe I will need to create a new PR for it, as I do not have access to your git-repo.

Reply:
https://github.com/okroener/bootstrap_grids/pull/1